pom.xml

```diff
@@ -517,6 +519,31 @@
           </execution>
         </executions>
       </plugin>
+
+      <plugin>
```

Comment:
sollen wir das noch in ein profile packen, sodass es nur beim Bauen von einem Docker-Container herangezogen wird?

Reply:
oder an eine dmp phase/goal hängen?